Modules/Vector/.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Remove this file from staging and add it to _.gitignore_ to prevent it from getting to others' computers (as Ron has already written)

Reply:
Can you relate to that comment, too?